### DISCUSSION TITLE
Add a GIF of the application's working to README

### DISCUSSION THREAD

**Discussion by Karthik-315** [subhoghoshX/laureate#46] [Ideas]
I was thinking of replacing the [demo image in README](https://github.com/subhoghoshX/laureate/blob/main/README.md#:~:text=Image%20of%20the%20application%20(Demo)) with a GIF. @subhoghoshX, let me know if you think its a good idea, I can implement it.

**Comment by subhoghoshX** [comment]
It can be really nice if implemented properly. Just wait this weekend. I might make some design changes.

**Reply by Karthik-315** [reply]
Cool dude. Let me know.

**Reply by subhoghoshX** [reply]
I think now is a good time to work on your idea. And sorry for being so late :grin:

**Reply by Karthik-315** [reply]
It's cool dude! I'll get right to it.

**Reply by Karthik-315** [reply]
PR sent. Let me know what you think of it.
